src/less/bootstrap/pagination.less

```diff
@@ -91,9 +91,9 @@
     //   &,
     //   &:hover,
     //   &:focus {
-    //     color: @oui-color-silver;
-    //     border-color: @oui-color-concrete;
-    //     background-color: @oui-color-concrete;
+    //     color: @g-300;
```

Comment:
Is this part couldn't be removed? (commented code)

Reply:
Good catch, I'll remove those parts